src/ui/public/notify/__tests__/notifier.js

```diff
@@ -229,16 +241,17 @@ describe('Notifier', function () {
       // destroy the default custom notification, avoid duplicate handling
       customNotification.clear();
 
-      const explicitLifetimeParams = _.defaults({ lifetime: 20000 }, customParams);
-      customNotification = notifier.custom(customText, explicitLifetimeParams);
+      const overrideParams = _.defaults({ lifetime: 20000, truncationLength: 1000 }, customParams);
+      customNotification = notifier.custom(customText, overrideParams);
 
       expect(customNotification).to.have.property('type', 'info'); // default
-      expect(customNotification).to.have.property('title', explicitLifetimeParams.title); // passed in
-      expect(customNotification).to.have.property('lifetime', explicitLifetimeParams.lifetime); // passed in
+      expect(customNotification).to.have.property('title', overrideParams.title); // passed in thru customParams
+      expect(customNotification).to.have.property('truncationLength', 1000); // passed in thru overrideParams
```

Comment:
Nit: Replace the `1000` with `overrideParams.truncationLength` so its consistent with the two assertions on the lines before and after this one?